contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +394,4 @@
 </form>
 <?php
 formFooter();
-?>
+?> 
```

Comment:
That would be my interpretation. Any files that are using echo or have html embedded do not require the omission of the closing tag. The reason for not including it in a file that is not supposed to output is because the parser treats any whitespace after `?>` as output and will echo it. Doesn't matter in a script that's getting printed anyway, but it does in libraries, like you're saying.